app/src/main/java/com/x8bit/bitwarden/ui/platform/components/badge/NotificationBadge.kt

```diff
@@ -0,0 +1,63 @@
+package com.x8bit.bitwarden.ui.platform.components.badge
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Badge
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.x8bit.bitwarden.ui.platform.theme.LocalNonMaterialColors
+
+/**
+ * Reusable component for displaying a notification badge.
+ *
+ * @param notificationCount numeric value to display in center of the badge.
+ */
+@Composable
+fun NotificationBadge(
+    modifier: Modifier = Modifier,
+    notificationCount: Int,
+    backgroundColor: Color = LocalNonMaterialColors.current.fingerprint,
+    contentColor: Color = MaterialTheme.colorScheme.onSecondary,
+) {
+    Badge(
+        content = {
+            if (notificationCount > 0) {
```

Comment:
Should the badge are if the number is 0 or below?

We could just remove this check, right?

Reply:
sure I can agree as we have not design for the dot w/o the text rn.

Reply:
wait well would we then want to move this text up above the "Badge" and put the visibility inside of here? Thinking about this as a standalone component it doesn't have the knowledge of when it shows or not?